config/routes.rb

```diff
@@ -33,6 +33,10 @@
                                    only: %i[index create] do
       collection do
         delete 'delete', to: 'user_notifications#delete'
+        get '/unsubscribe_from_all_notifications/:token', to: 'user_notifications#unsubscribe_from_all_notifications',
```

Comment:
Can you change these routes to use dashes instead of underscores?

i.e. `/unsubscribe-from-all-notifications/:token`, change the route below as well.